dispatcher/templates/member_expired.jinja2

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Filename should be membership_expired

Reply:
I don't find the corresponding entry anywhere in core 🤔

Reply:
We don't use it in the 'new' core, so I'm fine with removing it. We can add a new template when we add the functionality again